deepspeed/runtime/zero/partition_parameters.py

```diff
@@ -669,9 +669,13 @@ def get_model():
 
         # Remote device is the device where parameter partitions are stored
         # It can be same as local_device or it could be CPU or NVMe.
-        self.remote_device = self.local_device if remote_device is None else remote_device
-        self.pin_memory = pin_memory if (self.remote_device
-                                         == OffloadDeviceEnum.cpu) else False
+        self.remote_device = self.local_device if remote_device in [
+            None,
```

Comment:
Could we make the default of `remote_device` be `OffloadDeviceEnum.none`? Or do we need to support the option to pass `None`?

Reply:
Unfortunately, the caller could be outside deepspeed, e.g., Megatron, HF and other integrations, and `None` is a convenient default value for those cases.